generators/app/index.js

```diff
@@ -231,12 +237,28 @@ module.exports = generators.Base.extend({
       this.destinationPath('.babelrc')
     );
 
-    ['gulpfile.js', 'client', 'config', 'server', 'test'].forEach((f) => {
+    ['gulpfile.js', 'config', 'server', 'test'].forEach((f) => {
       this.fs.copy(
         this.templatePath(f),
         this.destinationPath(f)
       );
     });
+
```

Comment:
You can avoid copying the images and sw-register.js in the copyTpl using globOptions

```js
    this.fs.copyTpl(
      this.templatePath('client'),
      this.destinationPath('client'),
      { pwa: this.props.pwa },
      {}, // template options
      { // copy options
        globOptions: {
          ignore: [ "**/client/images/**", !this.props.pwa && "**/client/sw-register.js" ]
        }
      }
    );
```